hack/generate-yamls.sh

```diff
@@ -79,10 +80,12 @@ export KO_DOCKER_REPO
 cd "${YAML_REPO_ROOT}"
 
 echo "Building Knative Serving"
-ko resolve ${KO_YAML_FLAGS} -f config/ | "${LABEL_YAML_CMD[@]}" > "${SERVING_YAML}"
+ko resolve ${KO_YAML_FLAGS} -f config/ --selector networking.knative.dev/certificate-provider!=cert-manager | "${LABEL_YAML_CMD[@]}" > "${SERVING_YAML}"
```

Comment:
Can we do this for istio as well?

When we're done we should probably have:
1. `serving.yaml`: same as before
1. `serving-core.yaml`: just the irreplaceable bits from serving
1. `serving-istio.yaml`: just the istio controller
1. `serving-cert-manager.yaml`: just cert-manager controller

Reply:
Done